### ISSUE TITLE
No TV Shows  launching 

### ISSUE THREAD

**Issue by G10vanni74** [ABJC/ABJC-tvOS#49]
### Description of the Issue

I can browse my library of movies and TV, as well as the media pages that I find very elegant with the fanart in the background, I love it

I manage to watch some movies but no TV episode, I run the episode but nothing

For information, my server is on a DS218 play nas which runs with DSM 7

### tvOS Version

15

### ABJC Version

31

### ABJC TestFlight Build

the last 

### Additional information

I can now browse  my library of movies , I am no longer asked for a key API

The discovery of the server is simple

**Comment by noahkamara** [comment]
Happy to hear you like the design of the detail view 👯 

This issue is fixed in Build 32 (available as an alpha build now)

**Comment by G10vanni74** [comment]
Hello

I just noticed that some of my movies crash

I have movies in 4K HDR which pass and others not and the same for the H264.


I took a screenshot of the media information that does not launch.

I noticed one thing but I am not an expert, when the video is handled by the AVC1 and HVC1  codec, it crashes.

<img width="320" alt="2" src="https://user-images.githubusercontent.com/76523016/146229750-1df26606-bd19-4816-b1ca-2c2ff3ee14f1.png">
<img width="297" alt="1" src="https://user-images.githubusercontent.com/76523016/146229755-e9920766-62b8-4a15-9436-4fa424fbc75b.png">
